lib/ansible/module_utils/utm_utils.py

```diff
@@ -0,0 +1,170 @@
+# Copyright: (c) 2018, Johannes Brunswicker <johannes.brunswicker@gmail.com>
```

Comment:
Code under module_utils is under BSD licensed. Can you check if this possible ?

Reply:
Sure. Will adapt the license